packages/lite-insight/src/attribution/dimensionDrillDownAttribution.ts

```diff
@@ -0,0 +1,89 @@
+import { locatedInInterval, enumerateAllDimensionCombinationsByDFS } from './util';
+import type { Datum } from '../interface';
+import type {
+  DataConfig,
+  CompareInterval,
+  InfoType,
+  FluctInfo,
+  AttributionResult,
+  TreeDim,
+  DataLocation,
+} from './types';
+
+// Main function for dimension drill down attribution
+const attribute2MultiDimensions: (
+  sourceData: Datum[],
+  dimensions: string[],
+  measure: string,
+  fluctuationDim: string,
```

Comment:
you can use `fluctInfo: FluctInfo` to wrap the following three parameters together, for these three parameters are related.

Reply:
resolved